packages/jsii/lib/assembler.ts

```diff
@@ -226,7 +226,15 @@ export class Assembler implements Emitter {
     }
 
     async function _loadReadme(this: Assembler) {
-      const readmePath = path.join(this.projectInfo.projectRoot, 'README.md');
+      // Search for `README.md` in a case-insensitive way
+      const fileName =
+        (await fs.readdir(this.projectInfo.projectRoot)).find(
+          (file) => file.toLocaleLowerCase() === 'readme.md',
+        );
+      if (fileName == null) {
+        return undefined;
+      }
+      const readmePath = path.join(this.projectInfo.projectRoot, fileName);
       if (!(await fs.pathExists(readmePath))) {
         return undefined;
       }
```

Comment:
Are these lines still needed? Trying to think of a scenario where `readdir` matches a file but `pathExists` returns false.

Reply:
🤦🏻‍♂️